backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/services/GamesService.kt

```diff
@@ -0,0 +1,21 @@
+package pl.poznan.put.tsd.planningpoker.backend.services
+
+import org.springframework.stereotype.Service
+import pl.poznan.put.tsd.planningpoker.backend.model.Game
+import java.util.UUID
+import java.util.concurrent.ConcurrentHashMap
+
+@Service
+class GamesService {
```

Comment:
note: I will add tests for this class after we implement method to retrieve single game (probably together with `/join` endpoint)